mcos/mcos.py

```diff
@@ -37,3 +39,24 @@ def simulate_optimizations(
             'stdev': np.std(error_estimates[optimizer.name])
         } for optimizer in optimizers
     ]).set_index('optimizer')
+
+
+def simulate_optimization_from_price_history(
+        price_history: pd.DataFrame,
+        observation_name: str,
+        n_observations:int,
+        n_sims: int,
+        optimizers: List[AbstractOptimizer],
+        error_estimator: AbstractErrorEstimator,
+        covariance_transformers: List[AbstractCovarianceTransformer]):
+
+    mu, cov = convert_price_history(price_history)
+
+    if observation_name.lower() == "mucovledoitwolfobservationsimulator":
```

Comment:
Can we rename this to `mucovledoitwolf`? Truncate the other simulator's name as well.

Reply:
Could you also provide the list of valid simulator names in the README?